src/Provider/Pelias/README.md

```diff
@@ -21,6 +21,7 @@ You can run your own Pelias instance on your server, see [documentation](https:/
 For instance:
 
 - Geocode Earth ([Website](https://geocode.earth/) & [Provider](https://github.com/geocoder-php/geocode-earth-provider))
+- OpenRouteService ([Website](https://openrouteservice.org/) & [Provider](https://github.com/geocoder-php/open-route-service-provider))
```

Comment:
I would put openrouteservice (without dashes) since it is how they refer to it (see https://github.com/GIScience/openrouteservice)